<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BleAssistWidget</class>
 <widget class="QWidget" name="BleAssistWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BleAssistWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>蓝牙设置：</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>设备：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cb_devices"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btn_refresh">
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: transparent;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/icon/refresh</normaloff>:/icon/refresh</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_connect_status">
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="resource.qrc">:/icon/disc</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QPushButton" name="btn_connect">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>连接设备</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gb_connected">
       <property name="title">
        <string>已连接设备：</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_device_name">
          <property name="text">
           <string>名称：xxx</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_device_addr">
          <property name="text">
           <string>地址：xx:xx:xx</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" rowspan="4">
        <widget class="QTextEdit" name="edit_recv">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>HEX显示</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>加时间戳</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>清空接收</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1" colspan="2">
        <widget class="QCheckBox" name="checkBox_4">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>加回车换行</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" rowspan="2" colspan="2">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>清空发送</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QPushButton" name="btn_send">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QCheckBox" name="checkBox_3">
         <property name="text">
          <string>HEX发送</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" rowspan="6">
        <widget class="QTextEdit" name="edit_send">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>160</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
